test/e2e/helpers.js

```diff
@@ -33,6 +36,9 @@ const convertToHexValue = (val) => `0x${new BigNumber(val, 10).toString(16)}`;
 
 const convertETHToHexGwei = (eth) => convertToHexValue(eth * 10 ** 18);
 
+// Code from https://stackoverflow.com/questions/29183044
+const isInTest = typeof global.it === 'function';
```

Comment:
is this code not always run in a test?

Reply:
Surprisingly it's not!  The `benchmark` and `user-actions-benchmark` jobs in CircleCI will return `false` here.